### ISSUE TITLE
API: Possible 64-bit Integer Overflow in JavaScript Clients

### ISSUE THREAD

**Issue by cryptokat** [semuxproject/semux#729]
#### System information

Semux version: `v1.0.1`

#### Rationale 

Jackson [by default](https://github.com/FasterXML/jackson-databind/issues/911) serializes `Long` columns in plain numbers without considering the fact that JavaScript which is a popular client-side language, may not be able to deserialize 64-bit numbers correctly under some circumstances.

#### Demo of the Issue

Node.js demo which requests for an account's info then prints both of raw API response and deserialized json object:
```
> require('request-promise').get('http://user:pass@127.0.0.1:5171/get_account\?address\=0x541365fe0818ea0d2d7ab7f7bc79f719f5f72227&pretty=true').then(res => { console.log(res, JSON.parse(res)); });
> 
Raw:
{
  "success" : true,
  "result" : {
    "address" : "0x541365fe0818ea0d2d7ab7f7bc79f719f5f72227",
    "available" : 9399888811586101,
    "locked" : 300000000000000,
    "nonce" : 5111,
    "transactionCount" : 5121
  }
} 
Deserialized:
{ success: true,
  result: 
   { address: '0x541365fe0818ea0d2d7ab7f7bc79f719f5f72227',
     available: 9399888811586100,
     locked: 300000000000000,
     nonce: 5111,
     transactionCount: 5121 } }
```

The column of `result.available` is wrongly deserialized as `9399888811586100`

#### Proposed Fix

There are 2 proposed ways to fix this issue:

1. We may like to introduce an alternative `String` column for each `Long` column in API response. For example, to add a `available_str` column in string as an alternative of `available` column, as [Twitter has done](https://developer.twitter.com/en/docs/basics/twitter-ids) to its `id` column. In this way we can avoid breaking compatibility with existing 3rd-party applications. 
2. To return all numbers in `String`, which can be a breaking change

#### Credits

Thanks to @witoldsz for reporting this issue

**Comment by witoldsz** [comment]
I did not propose to use additional `…_str` fields, just pointed that Twitter had to to that in order to be backward compatible. It was a mistake from their side that they did not figured out this earlier.

I think it would be better if we introduced a braking change, it is probably still not too late to do it, considering everyone we know, using this API, would be informed early enough.

**Comment by cryptokat** [comment]
@witoldsz Sorry about the misquoting. I changed my statement above.

**Comment by cryptokat** [comment]
I am working on an optional configuration that allows API users to get numbers in string + a warning message about this issue during API startup.

**Comment by cryptokat** [comment]
See #730

**Comment by cryptokat** [comment]
resolved by #764
